logos/src/logos/main.py

```
@@ -3364,6 +3371,129 @@ async def logosnode_reconfigure_lane(data: LogosNodeReconfigureLaneRequest):
+    'needs calibration' but returns the full set instead of stopping at the
+    first hit. Sourced from configured_models so models the worker stripped
+    from capabilities_models (because they have no profile yet) are visible.
+    """
+    if _logosnode_facade is None:
+        return []
+    candidates = _logosnode_facade.get_configured_models(provider_id)
+    if not candidates:
+        candidates = _logosnode_facade.get_worker_capabilities(provider_id)
+    try:
+        profiles = _logosnode_facade.get_model_profiles(provider_id)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)